pkg/apis/planetscale/v2/vitessbackupstorage_types.go

```diff
@@ -67,6 +67,8 @@ type VitessBackupLocation struct {
 	GCS *GCSBackupLocation `json:"gcs,omitempty"`
 	// S3 specifies a backup location in Amazon S3.
 	S3 *S3BackupLocation `json:"s3,omitempty"`
+	// Azblob specifies a backup location in Azure Blob Storage.
```

Comment:
Run `make generate` to update generated code and docs.

Reply:
I get this error and I don't know how to convince it to get that dependency
```
operator-sdk-v0.10.0 generate k8s
Error: chdir /home/dk/go/src/planetscale.dev/vitess-operator/vendor/k8s.io/code-generator: no such file or directory
```

Reply:
and this 
```
> operator-sdk-v0.10.0 generate openapi --verbose
DEBU[0000] Debug logging is set                         
DEBU[0000] Running []string{"go", "build", "-o", "/home/dk/go/src/planetscale.dev/vitess-operator/build/_output/bin/openapi-gen", "./cmd/openapi-gen"} 
can't load package: package ./cmd/openapi-gen: cannot find package "." in:
	/home/dk/go/src/planetscale.dev/vitess-operator/vendor/k8s.io/kube-openapi/cmd/openapi-gen
Error: failed to exec []string{"go", "build", "-o", "/home/dk/go/src/planetscale.dev/vitess-operator/build/_output/bin/openapi-gen", "./cmd/openapi-gen"}: exit status 1
```

Reply:
Are you trying to run those outside the Makefile? If so, you need to `export GO111MODULE=on` yourself.

Reply:
With the Makefile I don't get the error message. they just Exit 1